src/lib/dev-environment/types.ts

```diff
@@ -74,9 +74,14 @@ export interface ConfigurationFileOptions {
 	'media-redirect-domain'?: string;
 	photon?: boolean;
 
+	meta?: ConfigurationFileMeta;
```

Comment:
Added this `meta` key for `ConfigurationFileOptions` to store information about the configuration file that doesn't affect the instance. This meta currently stores `configuration-path` which is used to show this configuration status line:

![configuration-file-meta](https://github.com/Automattic/vip-cli/assets/17870752/325ff33a-8663-4a23-af92-dc3482364987)